CHANGES.md

```diff
@@ -28,6 +28,8 @@ To be released.
  -  `Swarm<T>.PreloadAsync()` method and `Swarm<T>.StartAsync()` method became
     to take `preloadBlockDownloadFailed` event handler as an argument.
     [[#694]]
+ -  Moved `StoreExtension.ListAllStateReferences<T>()` method to
+    `IStore.ListAllStateReferences<T>()` method.  [[#701]]
```

Comment:
Isn't the destination `DefaultStore` (not `IStore`)?

Reply:
You're right. However, to add a `DefaultStore`, first, we need to add it to the `IStore` interface.

Reply:
Ah, okay. Then IMHO it's better to have two changes: removed an extension method, and added a new method into the interface.

Reply:
Added `Removed StoreExtension.ListAllStateReferences<T>() method.` to Backward-incompatible API changes and  `Added IStore.ListAllStateReferences<T>() method.` to Added APIs.  @longfin @dahlia

Reply:
I guess the changes you mentioned did not applied.

Reply:
Oh, it's missed while rebasing. Fixed.